app/packages/looker/src/elements/video.ts

```diff
@@ -736,24 +752,37 @@ export class VideoElement extends BaseElement<VideoState, HTMLVideoElement> {
 }
 
 export function withVideoLookerEvents(): () => Events<VideoState> {
-  return function () {
+  return () => {
+    let timeout: ReturnType<typeof setTimeout> = null;
     return {
       mouseenter: ({ update }) => {
         update(({ config: { thumbnail } }) => {
           if (thumbnail) {
+            timeout = setTimeout(() => {
+              update({
+                playing: true,
+                waitingToStream: false,
```

Comment:
Currently, a new frame stream is established as soon as mouseenter happens for video thumbnails. This is not ideal for dense data when scrolling in the grid. `waitingToStream` allows for an 500ms timeout